applet/src/radiostate.c

```diff
@@ -59,7 +59,8 @@ void rst_voice_lc_header(lc_t *lc)
         
         LOGR("cs %c %d->%d\n", grp_c, src, dst );
 
-        rst_voice_active = 1 ;    
+        rst_voice_active = 1 ;
+	rx_voice = 1 ;				// flag for new voice call received    
```

Comment:
rst_voice_active is being set during the whole reception of a voice call, using this would cause multiple lheard entries in the list for the whole call hold time during reception. rx_voice is set the same time, but is being reset immediately after the first lh logentry has been written to the display/buffer.